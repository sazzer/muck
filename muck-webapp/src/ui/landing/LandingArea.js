// @flow

import React from 'react';
import {Header, Image} from 'semantic-ui-react';
import {Interpolate} from 'react-i18next';

/** The Base URL to the Public files */
const publicUrl = process.env.PUBLIC_URL || '';

/**
 * React component to represent the Landing Page
 */
export default function LandingArea() {
    return (
        <div>
            <Header dividing as="h3">
                <Interpolate i18nKey="page.landing.title" />
            </Header>
            <Image src={`${publicUrl}/landingPage.jpg`} fluid />
            <p>
                <Interpolate i18nKey="page.landing.blurb" />
            </p>
        </div>
    )
}
