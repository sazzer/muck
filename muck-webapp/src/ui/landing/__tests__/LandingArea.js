// @flow

import React from 'react';
import {shallow} from 'enzyme';
import LandingArea from '../LandingArea';

/** Create the component to test */
function setup() {
    return {
        element: shallow(<LandingArea />)
    };
}

describe('LandingArea', () => {
    it('Renders correctly', () => {
        const { element } = setup();

        expect(element).toMatchSnapshot();
    });
});
