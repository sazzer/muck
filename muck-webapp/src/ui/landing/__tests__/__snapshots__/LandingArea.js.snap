// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingArea Renders correctly 1`] = `
<div>
  <Header
    as="h3"
    dividing={true}
  >
    <Interpolate
      className=""
      i18nKey="page.landing.title"
    />
  </Header>
  <Image
    as="img"
    fluid={true}
    src="/landingPage.jpg"
    ui={true}
  />
  <p>
    <Interpolate
      className=""
      i18nKey="page.landing.blurb"
    />
  </p>
</div>
`;
